package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "integration-ts-example",
+    "type": "module",
+    "version": "1.0.0",
```

Comment:
Start with 0.1.0 for the initial release.
Once the integration library reaches a stable v1 version, we can pumb the example versions as well.